docs/log-collector/README.md

```diff
@@ -1,26 +1,57 @@
 # k8s-triliovault Log Collector
-This is a python **pip** module that collects the information mainly yaml configuration and logs from k8s cluster for debugging k8s-triliovault application.
 
-## Requirements
-1. python >= 3.6
+Log collector let you define what you need to log and how to log it by collecting the the logs and events of Pod. Pod Logs can help you understand what is happening inside your application. The logs are particularly useful for debugging problems and monitoring cluster activity, alongside the metadata of all resources related to TrilioVault as either namespaced by providing namespaces name separated by comma or clustered from k8s cluster for debugging k8s-triliovault application. It also collects the CRDs yaml related to TVK and zip them.
 
-## How to use
-1. ```pip install k8s-triliovault-logcollector --extra-index-url https://pypi.fury.io/k8s-triliovault/``` 
-    
-2. ```log_collector.py```
+## Pre-requisites:
 
-Optional arguments: 
+1. krew - kubectl-plugin manager. Install from [here](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
+2. kubectl - kubernetes command-line tool. Install from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+
+**Installation, Upgrade, Removal of Plugins**:
+
+
+- Add TVK custom plugin index of krew:
+
+
+    kubectl krew index add tvk-plugins https://github.com/trilioData/tvk-plugins.git
+
+
+- Installation:
+
+
+    kubectl krew install tvk-plugins/tvk-log-collector
+
+
+- Upgrade:
+
+
+    kubectl krew upgrade
```

Comment:
update it to >>   kubectl krew upgrade tvk-log-collector

Reply:
resolved